apps/ledger-live-mobile/src/screens/Settings/Debug/Features/Swipper/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { Dimensions, StyleSheet, View } from "react-native";
+import { Gesture, GestureDetector } from "react-native-gesture-handler";
+import Animated, { runOnJS, useSharedValue, withSpring } from "react-native-reanimated";
+import Card from "./card";
+import { INITIAL_CARDS } from "./__mocks__/reviews";
+
+const { width, height } = Dimensions.get("window");
+const SWIPE_THRESHOLD_X = width * 0.3;
+const SWIPE_THRESHOLD_Y = height * 0.3;
+
+export default function SwipperScreen() {
+  const [cardIndex, setCardIndex] = useState(0);
+  const activeIndex = useSharedValue(0);
+  const swipeX = useSharedValue(0);
+  const swipeY = useSharedValue(0);
+
+  const cards = useMemo(() => {
+    const rotatedCards = [...INITIAL_CARDS];
+    const firstCards = rotatedCards.splice(0, cardIndex);
+    return [...rotatedCards, ...firstCards];
+  }, [cardIndex]);
+
+  const handleSwipeComplete = useCallback(() => {
+    runOnJS(setCardIndex)((prev: number) => (prev + 1) % INITIAL_CARDS.length);
```

Comment:
runOnJs already set so not needed here
```suggestion
    setCardIndex(prev => (prev + 1) % INITIAL_CARDS.length);
```